sorts/split_workload.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Tuple, Union
+
+
+def split_list(timings: List[Union[int, float, str]]) -> Tuple[List[Union[int, float]]]:
+    """
+
+    This algorithm is a brute-force search over (nearly) all 2^n
+    possible partitions and was created for educational purposes.
+    the asymptotic runtime of this code is: O(n * 2^n)
+
+    this is a case of the partition problem.
+    it accepts a multiset ( list ) of positive integers,
```

Comment:
It says here that the list must be of positive, but there is a working test including a list of negative numbers